src/mpl/include/mpl_base.h

```diff
@@ -98,6 +98,18 @@
 #define likely(x_)   (x_)
 #endif
 
+#ifdef HAVE_C11__STATIC_ASSERT
```

Comment:
MPL's config output is prefixed with `MPL_`, so I think this check needs to be modified.

> /* Define if C11 _Stattic_assert is supported. */
> #ifndef MPL_HAVE_C11__STATIC_ASSERT
> #define MPL_HAVE_C11__STATIC_ASSERT  1
> #endif